### ISSUE TITLE
OBS now supports Arch [community] repository

### ISSUE THREAD

**Issue by gojpdchx** [fusion809/fusion809.github.io#1]
Page affected: https://fusion809.github.io/how-to-create-archlinux-repository

I cannot find any news regarding the addition of `[community]` repository, but I can now specify `<path project="Arch:Community" repository="standard"/>`.
(e.g. By running command `osc meta prj home:yourusername -e`)

You may wish to try building a package on Open Build Service that depends on the `[community]` repository to verify that it works.

**Comment by fusion809** [comment]
I shall get round to testing this. I just have forgotten which packages in the AUR depend on community packages.

**Comment by fusion809** [comment]
Hooray I found a package to test it out on. Trizen, it has no AUR dependencies but depends on several community packages. Regrettably it doesn't build, due to missing dependencies found in the community repo. I added repositories for Arch Core, Extra and Community to my projects. Builds for all three failed for Trizen, two of which were due to missing dependencies. The Arch Community repo said its build state was broken. So for now I'm afraid I will not be amending my post.

**Comment by gojpdchx** [comment]
@fusion809 
I believe that you may have configured the repositories wrongly.

`icemon`, which has a make dependency of `docbook2x`, can be successfully built on OBS:
https://build.opensuse.org/package/show/home:user5664536:icemon/icemon

To add the [community] repository, you can:
1. Use `osc meta prj -e home:fusion809` and edit the content to something like:
```
<project name="home:user5664536:icemon">
  <title>icemon</title>
  <description>icemon</description>
  <person userid="user5664536" role="maintainer"/>
  <repository name="Arch">
    <path project="Arch:Core" repository="standard"/>
    <path project="Arch:Extra" repository="standard"/>
    <path project="Arch:Community" repository="standard"/>
    <path project="home:user5664536:icecream" repository="Arch"/>
    <arch>x86_64</arch>
  </repository>
</project>
```

2. Go to https://build.opensuse.org/repositories/home:fusion809 and delete all Arch repositories.
Then, click "Add repositories".
Find "Arch Core" and check it, then go back to https://build.opensuse.org/repositories/home:fusion809 . _(I need this step when using the online interface to edit repositories.)_
Now, click the respective "Edit repository" for that Arch repository you have just created.
Click "Add additional path to this repository".
In the text field for "Project", enter `Arch:E` and a suggestion `Arch:Extra` will be provided. Click on that suggestion. Click "Add path to repository Arch".
Repeat the last step but this time enter `Arch:C`. Two suggestions `Arch:Core` and `Arch:Community` will be provided. Click on `Arch:Community` then "Add path to repository Arch".

Afterwards wait a while or manually trigger a rebuild. You should now be able to use the packages from the [community] repository.

**Comment by fusion809** [comment]
Fixed in #dd6e00b.
